### PR TITLE
Update CLI to Docs Version 1.2

### PR DESCRIPTION
**Pull request by csamatas**

### CODE REVIEW

File: src/commands/conferences/create.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class conferencesCreate extends Command {
         waitUrl: flags.string({
             char: "w",
             description:
-                "If specified, a URL for the audio file that provides custom hold music for the Conference when it is in the populated state. Otherwise, FreeClimb uses a system default audio file. This is always fetched using HTTP GET and is fetched just once — when the Conference is created.",
+                "If specified, a URL for the audio file that provides custom hold music for the Conference when it is in the populated state. Otherwise, FreeClimb uses a system default audio file. This is always fetched using HTTP GET and is fetched just once &mdash; when the Conference is created.",
```
Comment (jblack-vail) [inline src/commands/conferences/create.ts:31]: Override to avoid HTML entity

File: src/commands/incoming-numbers/list.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export class incomingNumbersList extends Command {
         phoneNumber: flags.string({
             char: "p",
             description:
-                "Only show incoming phone number resources that match this PCRE-compatible  regular expression.",
+                "Only show incoming phone number resources that match this <a href=https://www.pcre.org/ target=blank>PCRE-compatible</a>  regular expression.",
```
Comment (jblack-vail) [inline src/commands/incoming-numbers/list.ts:14]: Override HTML link tag

**Review by jblack-vail — CHANGES_REQUESTED** [review]

File: src/commands/conferences/create.ts (hunk 1 above)
Comment (csamatas) [reply src/commands/conferences/create.ts]: common html entiities are now removed in generation of the schema

File: src/commands/incoming-numbers/list.ts (hunk 2 above)
Comment (csamatas) [reply src/commands/incoming-numbers/list.ts]: thanks for the catch anything encapsulated in <> are not removed in the generation of the schema

File: docs/calls.md [hunk 3]
```diff
@@ -120,20 +124,31 @@ OPTIONS
...
+      reduce latency when adding child calls to Conferences containing the parent Call. A call can only be used as a 
+      parent once the call is in progress or as an inbound call that is still ringing.  An outbound call is considered to 
+      be in progress once the outdialConnect or outdialApiConnect webhook is invoked.  An inbound call is ringing when the 
+      inbound webhook is invoked. If a callConnectUrl attribute is also included with the parentCallId in the request, 
+      this URL will be used as a replacement of the callConnectUrl originally assigned in the parent call.
+
+  -c, --callConnectUrl=callConnectUrl
+      The URL that FreeClimb should use to handle this phone call. If an applicationId or parentCallId have already been 
+      provided, this callConnectUrl attribute will be used as a replacement of the callConnectUrl originally assigned in 
+      the application or parent call. See applicationId v. callConnectUrl below.
```
Comment (apvail) [inline docs/calls.md:138]: > See applicationId v. callConnectUrl below.

???

**Review by apvail — CHANGES_REQUESTED** [review]
A lot of removal of `-n`/`--next` flags. Is this intended?

**Comment by apvail** [comment]
A lot of removal of -n/--next flags. Is this intended?

File: docs/calls.md (hunk 3 above)
Comment (apvail) [reply docs/calls.md:138]: I didn't see any section called any variation of "applicationId v. callConnectUrl" in these docs, so I'm confused by this sentence.

Comment (csamatas) [reply docs/calls.md]: i removed all sentences that have the words see below in them as long as they dont have the word note as well

**Review by apvail — APPROVED** [review]

**Review by jblack-vail — CHANGES_REQUESTED** [review]
I think that because this is a breaking change to many of the commands and their operations, that this should be made as a major release (e.g. 1.0.0). This would need to be added in the Changelog, ideally along with a description of all the changes, in order for the new automated deployment script to activate.

**Comment by csamatas** [comment]
> I think that because this is a breaking change to many of the commands and their operations, that this should be made as a major release (e.g. 1.0.0). This would need to be added in the Changelog, ideally along with a description of all the changes, in order for the new automated deployment script to activate.

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,23 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="0.3.0"></a>
+
+## [0.3.0] - 2021-4-07
```
Comment (jblack-vail) [inline CHANGELOG.md:14]:
```suggestion
## [0.3.0] - 2021-04-07
```

**Review by jblack-vail — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (csamatas) [reply CHANGELOG.md]: Fixed

**Pull request merged**